cmd/kritis/admission/main.go

```diff
@@ -65,8 +56,37 @@ func main() {
 		fmt.Println(version.Commit)
 		return
 	}
+
+	kritisConfigs, err := kritisconfig.KritisConfigs("default")
```

Comment:
This assumes users will deploy the config to the "default" namespace, so you might want to change it to `""` to get all configs in all namespaces!

Reply:
Good point, this is a leftover from debugging. Fixed.